app/controllers/applications_controller.rb

```diff
@@ -3,4 +3,23 @@ class ApplicationsController < ApplicationController
   def new
     @application = Application.new
   end
+
+  def create
+    user = User.find(params[:user_id])
+    application = user.applications.new(application_params)
+
+    if application.save
+      flash[:notice] = 'Application Submitted'
+      redirect_to '/dashboard'
+    else
+      flash[:notice] = 'Submission Failed'
```

Comment:
Look at that sad path
![](https://media.giphy.com/media/l3vR11Mr4XpqhtSHm/giphy.gif)